Run `nx release changelog` on Github action workflow could not determine Github repo slug

### Current Behavior

Throw error `Unable to create a Github release because the Github repo slug could not be determined` when run `nx release changelog` on Github action workflow

### Expected Behavior

Run `nx release changelog` on Github action workflow can determine repo slug and create release

### GitHub Repo

_No response_

### Steps to Reproduce

1.


### Nx Report

```shell
NX   Report complete - copy this into the issue template

   Node   : 18.18.2
   OS     : linux-x64
   pnpm   : 8.7.0
   
   nx                 : 17.0.2
   @nx/js             : 17.0.2
   @nx/jest           : 17.0.2
   @nx/linter         : 17.0.2
   @nx/eslint         : 17.0.2
   @nx/workspace      : 17.0.2
   @nx/devkit         : 17.0.2
   @nx/eslint-plugin  : 17.0.2
   @nrwl/tao          : 17.0.2
   typescript         : 5.1.6
```


### Failure Logs

```shell
> pnpm exec nx release changelog 0.4.0 --tagVersionPrefix="" --gitRemote origin

 >  NX   Generating a Github release for 0.4.0


 >  NX   Unable to create a Github release because the Github repo slug could not be determined.

   Please ensure you have a valid Github remote configured. You can run `git remote -v` to list your current remotes.

Error: Command failed: pnpm exec nx release changelog 0.4.0 --tagVersionPrefix="" --gitRemote origin
    at checkExecSyncError (node:child_process:890:11)
    at execSync (node:child_process:962:15)
    at /home/runner/work/sandox/sandox/tools/scripts/release.ts:74:13
    at step (/home/runner/work/sandox/sandox/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:195:27)
    at Object.next (/home/runner/work/sandox/sandox/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:176:57)
    at fulfilled (/home/runner/work/sandox/sandox/node_modules/.pnpm/tslib@2.6.2/node_modules/tslib/tslib.js:166:62) {
  status: 1,
  signal: null,
  output: [ null, null, null ],
  pid: 2185,
  stdout: null,
  stderr: null
}
 ELIFECYCLE  Command failed with exit code 1.
Error: Process completed with exit code 1.
```


### Package Manager Version

_No response_

### Operating System

- [ ] macOS
- [ ] Linux
- [ ] Windows
- [ ] Other (Please specify)

### Additional Information

_No response_

--- a/packages/nx/src/command-line/release/utils/github.ts
+++ b/packages/nx/src/command-line/release/utils/github.ts
@@ -38,7 +38,7 @@ export function getGitHubRepoSlug(remoteName = 'origin'): RepoSlug {
     }).trim();
 
     // Extract the 'user/repo' part from the URL
-    const regex = /github\.com[/:]([\w-]+\/[\w-]+)\.git/;
+    const regex = /github\.com[/:]([\w-]+\/[\w-]+)/;
     const match = remoteUrl.match(regex);
 
     if (match && match[1]) {

